doc/content/devices/best-practices/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Best Practices"
+description: ""
+weight:
+---
+
+This section contains best practices for building connected devices which use {{% tts %}} as a Network Server.
+
+<!--more-->
+
+LoRaWAN devices should always comply to the [LoRaWAN specification](https://www.lora-alliance.org/lorawan-for-developers), with special regard to minimizing unnecessary join requests and duty cycle limitations.
+
+The goal of these best practices is to optimize individual devices (especially for battery consumption) and the network as a whole (to reliably and efficiently serve more devices with the same number of gateways).
```

Comment:
```suggestion
The goal of these best practices is to optimize various aspects of individual device operation (with emphasis on minimizing power consumption) and the performance of the network as a whole (to reliably and efficiently serve more devices with the same number of gateways).
```

I think the explanation of network optimizations can also be rephrased to sound more professional. Perhaps, talk about scalability in regards to the gateway count?
Also, maybe a good idea to split this up in a proper paragraph and clearly state the properties we optimize for instead of putting it in parentheses?